app/_components/PrimaryButton/PrimaryButton.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link';
+import React from 'react';
+
+type PrimaryButtonPropType = {
+  to: string;
+  size: string;
```

Comment:
`large`, `small`으로 정한거면 타입을 더 제한하면 좋을 것 같아요..!

```suggestion
  size: 'large' | 'small';
```

Reply:
넵 저도 타입을 최대한 제한해 사용하기 편하게 하고 싶어 2개로 줄인 것도 있어서 종길님 말씀처럼 처리해주시면 좋을 것 같습니다~!